.github/ct-lint.yaml

```diff
@@ -15,6 +15,14 @@ chart-dirs:
   - charts
 
 chart-repos:
-  - kube-state-metrics=https://kubernetes.github.io/kube-state-metrics
-  - pixie-operator=https://pixie-operator-charts.storage.googleapis.com
+  - newrelic-helm-charts=https://newrelic.github.io/helm-charts
+  - newrelic-infrastructure=https://newrelic.github.io/nri-kubernetes
+  - nri-prometheus=https://newrelic.github.io/nri-prometheus
+  - nri-metadata-injection=https://newrelic.github.io/k8s-metadata-injection
   - newrelic-k8s-metrics-adapter=https://newrelic.github.io/newrelic-k8s-metrics-adapter
+  - kube-state-metrics=https://kubernetes.github.io/kube-state-metrics
+  - nri-kube-events=https://newrelic.github.io/nri-kube-events
+  # - newrelic-logging= Yet to be moved/deprecated
```

Comment:
I would add these once they are moved/deprecated

Reply:
Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)